### PR TITLE
Update npm scripts

### PR DESCRIPTION
**Pull request by jfurrow**
Update the npm scripts as discussed in #79

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,11 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "livereload": "NODE_ENV='development' ./node_modules/.bin/gulp livereload",
-    "start": "npm prune && npm install && ./node_modules/.bin/gulp dist && ./node_modules/.bin/nodemon ./server/bin/www"
+    "preinstall": "npm prune",
+    "prestart": "./node_modules/.bin/gulp dist",
+    "start": "./node_modules/.bin/nodemon ./server/bin/www",
+    "start:production": "node ./server/bin/www",
+    "livereload": "NODE_ENV='development' ./node_modules/.bin/gulp livereload"
```
Comment (serut) [inline package.json]: start:watch is more commun

**Comment by serut** [comment]
Pre-Requisites 
- add : if you are want to use docker, rtorrent needs to open the XMLRPC entry point to the network. Ensure that you put `0.0.0.0` in the config of `.rtorrent.rc`

Starting the Server 
- rename to Starting the Server on dev
- add 0. npm install
- change 1. command to `npm run start:watch`

Add Starting the server on production:
- npm install --production (no devDependencies)
- npm run start (no nodemon => no watcher)

Otherwise it's a good improvement ! :+1:

File: package.json (hunk 1 above)
Comment (jfurrow) [inline package.json]: I like it. Done!

**Comment by jfurrow** [comment]
@serut I structured the readme a little differently, but it should cover all of your concerns! Thanks for the suggestions.

**Comment by serut** [comment]
You may want to use webpack because it can watch, reload, create the production build... Instead of gulp. Your readme is perfect by the way

**Comment by jfurrow** [comment]
I've used Webpack's watch system in the past and wasn't pleased with its performance, but I'll check it out again in the future for sure.

**Comment by serut** [comment]
Webpack saves files on RAM so its quite good. And when I said webpack instead of gulp maybe I made a mistake and webpack can uses gulp. Dont know for sure but there is a lot of functunality with webpack

**Comment by jfurrow** [comment]
I'm actually using Webpack with Gulp already, just not making use of Webpack's watch feature as I found it slow in the past. It's certainly possible I didn't have it configured optimally, but I am very happy with the speed of the current setup.

**Pull request merged**